app/assets/stylesheets/components/_cards_my_infections.scss

```diff
@@ -0,0 +1,110 @@
+$past-infection-color-active: hsla(120, 0, 70, 1);
+$past-infection-color-cancelled: hsla(360, 0, 70, 1);
+$current-infection-color-active: hsla(120, 100, 50, 1);
+$current-infection-color-cancelled: hsla(360, 100, 50, 1);
+$future-infection-color-active: hsla(120, 100, 50, 0.3);
+$future-infection-color-cancelled: hsla(360, 100, 50, 0.3);
+
+.past.card-active {
+  background-color: $past-infection-color-active;
```

Comment:
@Kimon-Haars can past infection be active?

Reply:
with my logic they can, in the sense that they were not cancelled / actually happened.

Reply:
oh, I understand now, so basically it is not active that this will happen but in a way that it happened. all good then